tests/k8s/tests/001-policy-enforcement.sh

```diff
@@ -295,9 +295,13 @@ check_endpoints_policy_disabled ${NUM_ENDPOINTS} ${CILIUM_POD_1}
 # Test 18: check policy enforcement after malformed policy is created and deleted
 # All endpoints should have policy enforcement enabled
 log " Test 18: check policy enforcement after malformed policy is created and deleted. All endpoints should have policy enforcement enabled"
+set +e
```

Comment:
We expect this addition of malformed policy to fail, so we don't want the test to fail when we try to add it.